qubes/vm/qubesvm.py

```diff
@@ -660,6 +713,24 @@ class QubesVM(qubes.vm.mix.net.NetVMMixin, qubes.vm.BaseVM):
     #
     # properties loaded from XML
     #
+    active_bootmode = qubes.property(
```

Comment:
shouldn't this be called just "bootmode"?

Reply:
I guess that would be OK. I liked `active` personally but it's probably not necessary.

Reply:
Renamed.